shoop/core/order_creator/_source.py

```diff
@@ -404,6 +411,35 @@ def get_validation_errors(self):
             for error in payment_method.get_validation_errors(source=self):
                 yield error
 
+        for supplier in self._get_suppliers():
+            for product, quantity in iteritems(self._get_products_and_quantities(supplier)):
+                shop_product = product.get_shop_instance(shop=self.shop)
+                if not shop_product:
```

Comment:
IIRC get_shop_instance raises if such shop product does not exists, so this is wrong.